lighthouse-cli/bin.js

```diff
@@ -94,34 +95,35 @@ if (cliFlags.extraHeaders) {
 /**
  * @return {Promise<LH.RunnerResult|void>}
  */
-function run() {
-  return Promise.resolve()
-    .then(_ => {
-      // By default, cliFlags.enableErrorReporting is undefined so the user is
-      // prompted. This can be overriden with an explicit flag or by the cached
-      // answer returned by askPermission().
-      if (typeof cliFlags.enableErrorReporting === 'undefined') {
-        return askPermission().then(answer => {
-          cliFlags.enableErrorReporting = answer;
-        });
-      }
-    })
-    .then(_ => {
-      Sentry.init({
-        url,
-        flags: cliFlags,
-        environmentData: {
-          name: 'redacted', // prevent sentry from using hostname
-          environment: isDev() ? 'development' : 'production',
-          release: pkg.version,
-          tags: {
-            channel: 'cli',
-          },
-        },
-      });
+async function run() {
+  if (cliFlags.printConfig) {
+    const config = generateConfig(cliFlags, configJson);
+    process.stdout.write(config.print());
+    return;
+  }
 
-      return runLighthouse(url, cliFlags, config);
+  // By default, cliFlags.enableErrorReporting is undefined so the user is
+  // prompted. This can be overriden with an explicit flag or by the cached
+  // answer returned by askPermission().
+  if (typeof cliFlags.enableErrorReporting === 'undefined') {
+    cliFlags.enableErrorReporting = await askPermission();
+  }
+  if (cliFlags.enableErrorReporting) {
```

Comment:
this check isn't strictly necessary (`Sentry.init` itself checks for `enableErrorReporting`), but added this just to reassure anyone reading this file and not digging in farther